src/software/firmware/includes/parameters.h

```diff
@@ -29,11 +29,11 @@
 
 // Minimum & maximum execution parameters
 #define CONST_MAX_PEAK_PRESSURE 700u     // arbitrary [mmH2O]
-#define CONST_MIN_PEAK_PRESSURE 100u     // arbitrary [mmH2O]
+#define CONST_MIN_PEAK_PRESSURE 50u     // arbitrary [mmH2O]
 #define CONST_MAX_PLATEAU_PRESSURE 400u  // PP MAX ARDS = 300 [mmH2O]
-#define CONST_MIN_PLATEAU_PRESSURE 100u  // arbitrary [mmH2O]
+#define CONST_MIN_PLATEAU_PRESSURE 30u  // arbitrary [mmH2O]
 #define CONST_MAX_PEEP_PRESSURE 300u     // PP MAX = 300, or PEEP < PP [mmH2O]
-#define CONST_MIN_PEEP_PRESSURE 50u      // arbitrary but > 0 [mmH2O]
+#define CONST_MIN_PEEP_PRESSURE 10u      // arbitrary but > 0 [mmH2O]
```

Comment:
I'm not sure it's a good idea to authorize a PEEP under 2cmH2O. Alarm 11 (RCM-SW-2) is about having a pressure  below 2cmH2O.

@dsferruzza @cmdm what are you though on this?